targets/wasm_exec.js

```diff
@@ -298,9 +298,22 @@
 
 					// func finalizeRef(v ref)
 					"syscall/js.finalizeRef": (v_ref) => {
-						// Note: TinyGo does not support finalizers so this should never be
-						// called.
-						console.error('syscall/js.finalizeRef not implemented');
+						// Note: TinyGo does not support finalizers so this is only called
+						// for one specific case, by js.go:jsString. and can/might leak memory.
+						const id = mem().getUint32(unboxValue(v_ref), true);
+						// Note that this if is so far seemingly never true. Someone should investigate why.
+						if (this._goRefCounts?.[id] !== undefined) {
```

Comment:
The `unboxValue` returns the string as a byte array. It leads to `id` being `0` in my case.
So I looked around, and the `unboxValue` contained an interesting part of code:
```
const id = v_ref & 0xffffffffn;
return this._values[id];
```

So I tried this:
```suggestion
						// magicValue taken from `unboxValue` function.
						const magicValue = 0xffffffffn;
						const id = v_ref & magicValue;
						// Note that this if is so far seemingly never true. Someone should investigate why.
						if (this._goRefCounts?.[id] !== undefined) {
```
And when I checked `this._ids.size` before and after the `delete`, it really deleted something :D Can it be it? Is it working? Like, I have no idea what I'm doing and what the hex number is for.